airflow-core/src/airflow/ui/src/router.tsx

```diff
@@ -176,6 +176,15 @@ export const routerConfig = [
         element: <GroupTaskInstance />,
         path: "dags/:dagId/runs/:runId/tasks/group/:groupId",
       },
+      {
+        children: [
+          { element: <TaskOverview />, index: true },
+          { element: <TaskInstances />, path: "task_instances" },
+          { element: <Events />, path: "events" },
```

Comment:
Let's drop this since I don't think we can properly filter audit log events by group